### DISCUSSION TITLE
Add LongPress event

### DISCUSSION THREAD

**Discussion by criss02-cs** [dotnet/maui#17618] [Ideas]
hi everyone, I was developing my app with .NET MAUI and I noticed something, the possibility of managing the event of long pressure on a button, or in any other control, is missing, I solved it by creating a custom button that implements this functionality however I believe it should be included as a basic functionality since it is used a lot nowadays

**Comment by dartasen** [comment]
You can now also use [https://learn.microsoft.com/en-us/dotnet/communitytoolkit/maui/behaviors/touch-behavior?tabs=toucheffect-xaml%2Ctouchbehavior-xaml#handling-long-press-interaction](https://learn.microsoft.com/fr-fr/dotnet/communitytoolkit/maui/behaviors/touch-behavior?tabs=toucheffect-xaml%2Ctouchbehavior-xaml) as another solution
